oneShow.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TV Show Episode Guide</title>
+    <link href="oneShow.css" rel="stylesheet" />
+    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&family=Sorts+Mill+Goudy:ital@0;1&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
```

Comment:
this is quite an amazing way to separate and make the code more modular. I was struggling with this part.